content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment.
+ 
+
+\begin{itemize}
+
+\item
+In a thread-compliant implementation, an \openshmem{} \ac{PE} is multithreaded. 
+Each thread can issue \openshmem{} calls; however the threads
```

Comment:
I also think this mixes up two points. That a PE is multithreaded is an _application_ characteristic, but I think the point you're trying to make about thread compliance is an _implementation_ characteristic.